assets/src/components/parts/janus-input-number/InputNumberAuthor.tsx

```diff
@@ -20,8 +20,18 @@ const InputNumberAuthor: React.FC<AuthorPartComponentProps<InputNumberModel>> =
   }, []);
 
   return (
-    <div style={styles}>
-      <p>Input Number</p>
+    <div className={`number-input`} style={styles}>
+      <label htmlFor={`${id}-number-input`}>
+        {showLabel && label ? label : <span>&nbsp;</span>}
+      </label>
+      <input
```

Comment:
input number has a few more properties that can affect rendering no? like whether or not to show the number arrows